PublicAPI/Microsoft.ApplicationInsights.dll/net45/PublicAPI.Shipped.txt

```diff
@@ -108,6 +113,7 @@ Microsoft.ApplicationInsights.DataContracts.ExceptionTelemetry.ExceptionTelemetr
 Microsoft.ApplicationInsights.DataContracts.ExceptionTelemetry.ExceptionTelemetry(System.Exception exception) -> void
 Microsoft.ApplicationInsights.DataContracts.ExceptionTelemetry.HandledAt.get -> Microsoft.ApplicationInsights.DataContracts.ExceptionHandledAt
 Microsoft.ApplicationInsights.DataContracts.ExceptionTelemetry.HandledAt.set -> void
+Microsoft.ApplicationInsights.DataContracts.ExceptionTelemetry.ExceptionDetailsInfoList.get -> System.Collections.Generic.IReadOnlyList<Microsoft.ApplicationInsights.DataContracts.ExceptionDetailsInfo>
```

Comment:
Consider referencing only the first ExceptionDeails object that will have it's own children collection

Reply:
sorry, i didn't get what you are suggesting. Are you saying to have `IList<ExceptionDetails>` inside `ExceptionDetailsInfo` class? If yes, I didn't want it that way because then how would user change type and message of individual `ExceptionDetails`?

---
In reply to: [203866174](https://github.com/Microsoft/ApplicationInsights-dotnet/pull/865#discussion_r203866174) [](ancestors = 203866174)